kore/src/main/java/org/kframework/builtin/KLabels.java

```diff
@@ -42,6 +42,7 @@
     public static final KLabel SetItem = KLabel("SetItem");
     public static final KLabel DotMap = KLabel(".Map");
     public static final KLabel List = KLabel("_List_");
+    public static final KLabel DotSet = KLabel(".Map");
```

Comment:
`.Set` or `.Map`?

Reply:
BTW, the order of these constants makes little sense.

Reply:
Thanks for catching this!  And abput the order, I agree, but it's not like it was very ordered before :-)